### PR TITLE
Add Presence | CloudFM

### PR DESCRIPTION
**Pull request by marinofranz**
CloudFM_Presence

### CODE REVIEW

File: websites/C/CloudFM/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        }
+    ],
+    "url": ["cloudfm.xyz", "www.cloudfm.xyz"],
+    "description": {
+        "en": "Welcome to CloudFM! We play the latest music in various categories."
+    },
+    "service": "CloudFM",
+    "version": "1.0.0",
+    "logo": "https://dualfm.is-inside.me/c6rbC8cp.png",
+    "thumbnail": "https://dualfm.is-inside.me/Wr5pZXni.png",
```
Comment (Bas950) [inline websites/C/CloudFM/dist/metadata.json:19]: upload to imgur

Comment (marinofranz) [reply websites/C/CloudFM/dist/metadata.json:19]: fixing deepscan, and uploading to imgur

File: websites/C/CloudFM/dist/metadata.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
     },
     "service": "CloudFM",
     "version": "1.0.0",
-    "logo": "https://dualfm.is-inside.me/c6rbC8cp.png",
-    "thumbnail": "https://dualfm.is-inside.me/Wr5pZXni.png",
+    "logo": "https://imgur.com/RhJ2JvW.png",
+    "thumbnail": "https://imgur.com/2qAS8bw.png",
```
Comment (Bas950) [inline websites/C/CloudFM/dist/metadata.json:19]: change them to the i.imgur links

Comment (marinofranz) [reply websites/C/CloudFM/dist/metadata.json]: fixed

**Pull request merged**